### PR TITLE
metal3-dev: Copy ironic creds for local bmo

### PR DESCRIPTION
**Pull request by zaneb**

### CODE REVIEW

File: metal3-dev/local-bmo.sh [hunk 1]
```diff
@@ -71,6 +71,26 @@ for var in IRONIC_ENDPOINT IRONIC_INSPECTOR_ENDPOINT DEPLOY_KERNEL_URL DEPLOY_RA
...
+get_creds() {
+    local svc=$1
+    local cred_dir="${auth_dir}/${svc}"
+    local secret="metal3-${svc}-password"
+    if oc get -n openshift-machine-api secret ${secret} -o name >/dev/null; then
+        if [ ! -d ${auth_dir} ]; then
+            sudo mkdir -p "${auth_dir}"
+            sudo chown -R $USER:$USER "${auth_dir}/.."
+        fi
+        mkdir "${cred_dir}"
```
Comment (andfasano) [inline metal3-dev/local-bmo.sh:84]: `-p` requires also here

Comment (zaneb) [reply metal3-dev/local-bmo.sh:84]: No it isn't.

Comment (andfasano) [reply metal3-dev/local-bmo.sh:84]: Not sure if I'm missing something, but when I ran it twice (from a clean env, without `/opt/metal3`) the first time it's ok but the second fails with a `mkdir: cannot create directory ‘/opt/metal3/auth/ironic’: File exists`

Comment (zaneb) [reply metal3-dev/local-bmo.sh:84]: Ah, OK. The parent directory always exists, but `-p` also suppresses the error if the cred_dir itself exists. I'll fix it up, thanks.

**Pull request merged**